README.md

```diff
@@ -1,2 +1,52 @@
 # Microverse-Leader-Board
 Project from module 2 week 4
+
+This branch is the initial stage of the Leader board Api Project where i have build the basic structure for it using webpack.
+
+# Screenshot
+
+![Leader-Board](https://user-images.githubusercontent.com/70577783/150947789-f1dcb4ae-4ae7-48f5-95d4-38b0abbf61c7.png)
+
+# Built with
+This project uses the following open source packages:
+
+- HTML
+- CSS
+- node
+- Weback
+- Bootstrap
+- Javascript
+
+# Author
+
+GitHub [@AkshithaReddy1899](https://github.com)
+
+Twitter [@Akshitha Reddy](https://twitter.com)
+
+Codepen [@Akshitha Reddy](https://codepen.io/Akshitha_Reddy)
+
+
+
+# Getting Started
+
+To clone this repo: You need to have git installed in your system.
+
+- git clone the repo in your terminal by: `gh repo clone AkshithaReddy1899/Microverse-Leader-Board`
+- navigate to 
+`cd into the folder`
+
```

Comment:
- [ ] [OPTIONAL] Please include `npm install` as it is very crucial in users setting up your work in their local environment to install and read the packages and their dependencies.